lib/src/git.rs

```diff
@@ -68,11 +68,32 @@ pub fn import_refs(
     git_repo: &git2::Repository,
     git_settings: &GitSettings,
 ) -> Result<(), GitImportError> {
+    import_some_refs(mut_repo, git_repo, git_settings, |_| true)
+}
+
+/// Reflect changes made in the underlying Git repo in the Jujutsu repo.
+/// If `git_ref_filter` is defined, only branches whose git full reference name
```

Comment:
nit: Looks like the "is defined" part of this comment is a bit stale (I'm guessing `git_ref_filter` used to be an `Option`)

Reply:
Good catch.